docs/src/main/sphinx/release/release-382.md

```diff
@@ -36,6 +36,7 @@
   reading ORC data that contains Bloom filters. ({issue}`11732`)
 * Rename the `delete_orphan_files` table procedure to `remove_orphan_files`. ({issue}`12468`)
 * Improve query performance of reads after `DELETE` removes all rows from a file. ({issue}`12197`)
+* Remove HiveConfig and add support for the relevant configuration properties
```

Comment:
`HiveConfig` is internal class name and general users won't understand it. Could you reword the sentence? 
Also, it would be better to add ``` ({issue}`xxx`) ```

Reply:
Good point, reworked and added here https://github.com/trinodb/trino/issues/12556